test/acceptance/features/collection/step_definitions/collection.js

```
@@ -53,50 +71,43 @@ defineSupportCode(({ When, Then }) => {
   })
 
   Then(/^I can view the (.+) in the collection$/, async function (entityType, dataTable) {
-    const entityHeaderName = get(this.state, `${camelCase(entityType)}.header`)
+    const entityHeading = get(this.state, `${camelCase(entityType)}.heading`)
 
     await Collection
       .section.firstCollectionItem
       .waitForElementPresent('@header')
-      .assert.containsText('@header', entityHeaderName)
+      .assert.containsText('@header', entityHeading)
 
-    for (let row of dataTable.hashes()) {
-      const element = Collection.getSelectorForMetaListItem(row.label)
-      const expectedText = get(this.state, row.statePath)
+    for (const row of dataTable.hashes()) {
+      const metaListValueElement = Collection.getSelectorForMetaListItemValue(row.text)
+      const expectedMetaListText = get(this.state, row.expect)
 
-      if (expectedText) {
+      if (expectedMetaListText) {
```

We can remove this `if` and let it blow up.